.travis.yml

```diff
@@ -19,6 +19,8 @@ before_install:
   - sudo add-apt-repository -y ppa:staticfloat/juliareleases
   - sudo apt-get update
   - sudo apt-get install julia
+  - sudo apt-get install cabal-install
+  - cabal update
```

Comment:
why do we need this?
It is a very expensive command to run on CI, consuming over a minute.